client/src/main/java/org/ehrbase/client/openehrclient/defaultrestclient/DefaultRestClient.java

```diff
@@ -159,6 +159,24 @@ protected HttpResponse internalPost(
         return response;
     }
 
+    protected HttpResponse internalPut(
```

Comment:
use this in httpPut line 184

Reply:
I would recommend to use interalPut from httpPut because the method httpPut is using more header which are currently not needed for the stored query PUT